var/spack/repos/builtin/packages/spectrum-mpi/package.py

```diff
@@ -0,0 +1,57 @@
+from spack import *
+
+class Spectrum(Package):
+    """IBM MPI implementation from Spectrum MPI."""
+
+    homepage = "http://www.example.com"
```

Comment:
You can use http://www-03.ibm.com/systems/spectrum-computing/products/mpi/ for the homepage.